### ISSUE TITLE
Flag Sweden named incorrectly

### ISSUE THREAD

**Issue by lck-git** [cmsimple-xh/cmsimple-xh#584]
Since XH 1.7.0 the flag for Sweden is [sv.gif](https://github.com/cmsimple-xh/cmsimple-xh/blob/5a683164e3eb2fdee2dc8e052d72638011c7d6d7/userfiles/images/flags/sv.gif), it should be se.gif. See also in the [forum](https://cmsimpleforum.com/viewtopic.php?f=10&t=19816&p=88050#p88035).

**Comment by cmb69** [comment]
> Since XH 1.7.0 the flag for Sweden is [sv.gif](https://github.com/cmsimple-xh/cmsimple-xh/blob/5a683164e3eb2fdee2dc8e052d72638011c7d6d7/userfiles/images/flags/sv.gif), it should be se.gif

Nope. We're using the [ISO639-1 languages codes](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=sv) for the language menu, and the flag file should be named accordingly. Otherwise, we would need to rename da.gif to dk.gif as well (and likely some others).

**Comment by olape-git** [comment]
Oha, :-(
I wasn't aware of that.

**Comment by cmb69** [comment]
I don't see a big problem with this, though. Renaming a file, if someone needs to, isn't hard.

**Comment by olape-git** [comment]
But we should still correct it again.
However, I have to admit that if I had been supposed to put on Swedish, I would certainly have done it wrong.

I could not claim to know now that we use ISO639-1 languages codes.
I also don't know where this is documented.

**Comment by olape-git** [comment]
> See https://wiki.cmsimple-xh.org/?for-users/working-with-the-cms/multilingual-website&search=language:
> 
> > Rename this to the two-character ISO 639-1 language code for the desired language.

🤦‍♂️😒💩



> and there are solutions for this, e.g. https://cmsimpleforum.com/viewtopic.php?f=41&t=21038. :)

and here too ;-)

https://olaf.penschke.net/?CMSimple_XH/Plugins/Polyglot_XH

> Das Sprachmenü kann in Form des ISO-Codes ausgegeben werden.
